tests/robots/test_robot.py

```diff
@@ -1,14 +1,17 @@
+import copy
 import os
 import re
 
 import pytest
+from compas.datastructures import Mesh
 from compas.geometry import Frame
 from compas.robots import RobotModel
 
+import compas_fab
 from compas_fab.backends.interfaces import ClientInterface
 from compas_fab.backends.interfaces import InverseKinematics
 from compas_fab.backends.interfaces import PlannerInterface
-from compas_fab.robots import Robot
+from compas_fab.robots import Robot, Tool
```

Comment:
we always do single line imports:

```suggestion
from compas_fab.robots import Robot
from compas_fab.robots import Tool
```